src/main/java/TinhS.java

```diff
@@ -0,0 +1,11 @@
+public class TinhS {
+    public String S(long n){
+        double s=2;
+        long giaithua=1;
+        for(int i=2; i<=n; i++){
+            giaithua*=i;
+            s+=1.0/giaithua;
+        }
+        return String.valueOf(s);
```

Comment:
Phải trả về double

Reply:
Dạ trả về double bên thì bên test cái hàm assertEquals nó ko cho so double, double ạ

Reply:
Use this https://junit.org/junit4/javadoc/4.12/org/junit/Assert.html#assertEquals(double,%20double,%20double)

Reply:
Done